docs/modules/user-guide/partials/proc_adding-kubernetes-or-openshift-component-to-a-devfile.adoc

```diff
@@ -1,4 +1,4 @@
-[id="proc_adding-kubernetes-component-to-a-devfile_{context}"]
+[id="proc_adding-kubernetes--or-openshift-component-to-a-devfile_{context}"]
 = Adding a `kubernetes` or `openshift` component to a devfile
```

Comment:
Does devfiles use monospaced formatting in titles? In the OpenShift docs, I believe this is not allowed.

Reply:
See "Do not use backticks or other markup in assembly or module headings" in [Assembly/Module titles and section headings](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings).